src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -705,7 +705,7 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
         if (!handled)
         {
             // _TrySendKeyEvent pretends it didn't handle F7 for some unknown reason.
-            (void)_TrySendKeyEvent(VK_F7, 0, modifiers);
+            (void)_TrySendKeyEvent(VK_F7, 0, modifiers, true);

```

Comment:
do we need to emit an Up for F7 too?

Reply:
Probably? Though for this hack it seems fine, but I'll do it anyways